app/javascript/components/CurriculaSearch.js

```diff
@@ -0,0 +1,151 @@
+        return standardsJsonArray;
+    };
+
+    render() {
+        const { classes } = this.props;
+        const data = this.getCurriculaData();
+        const empty = (data.length == 0);
+        if (!empty) {
+            if (this.state.checkboxbools.length == 0) {
+                data.forEach((standard) => {
+                    this.state.checkboxbools.push({
+                        id: standard.id,
+                        description: standard.description,
+                        checked: false
+                    })
+                })
+            }
+        }
+        
+
+
+        return (
+            <React.Fragment>
+                <CssBaseline />
+                <div className={classes.root2}>
+                    <main className={classes.layout2}>
+
+                        <Paper className={classes.curriculaTitleHeader}>
+                            <div className={classes.courseName}>
+                                <Typography component="h1" className={classes.newFont} variant="h3" color="inherit" gutterBottom>
+                                    CurriculaHub
+                    </Typography>
+                                <Typography variant="h5" className={classes.newFont} color="inherit" paragraph>
+                                    A one-stop platform for you to explore, adapt, add, and discuss open-source curricula
+                    </Typography>
+                            </div>
+                        </Paper>
+
+                        <div>
+                            {empty &&
+                                <h2>
+                                    We don't currently have any curricula for your criteria. Stay tuned for an update!
+                                </h2>
+                             }
+                         </div>
+
+                            {!empty && 
+                                this.state.checkboxbools.map((checkbox) =>
+                                    <div>
+                                        <FormControlLabel control={<Checkbox value={checkbox.id} onChange={this.handleInputChange} checked={checkbox.checked} />} label={checkbox.id + ": " + checkbox.description} />
+                                    </div>
+                                )
+                            }
+                        
+
+                        <div>
+                            {!empty &&
+                                <Link to={{
+                                    pathname: "/results/" + this.props.match.params.age + "/" + this.props.match.params.topic,
+                                    state: {
+                                        checkboxdata: this.state.checkboxbools
+                                    }
+                                }}>
+                                    <Button variant="contained" color="secondary" className={classes.button}>
+                                        <SearchIcon />
```

Comment:
search is cool! except i expected that i would get the intersection, not the union of my checked boxes